src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/OutConsole.cs

```diff
@@ -43,7 +43,7 @@ protected override void ProcessRecord()
     /// default sink (display to console screen)
     /// </summary>
     [Cmdlet(VerbsData.Out, "Default", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113362", RemotingCapability = RemotingCapability.None)]
-    public class OutDefaultCommand : FrontEndCommandBase
+    public class OutDefaultCommand : FrontEndCommandBase, IDisposable
```

Comment:
That was a leftover from before I realized FrontEndCommandBase implements IDisposable.  Will remove.